examples/fabric/reinforcement_learning/README.md

```diff
@@ -82,3 +84,36 @@ lightning run model \
   --num-steps 2048 \
   --seed 1
 ```
+
+## Example 2 - Environment decoupled from the agent
+
+In this example we have gone even further leveraging the flexibility offered by [Fabric](https://lightning.ai/docs/fabric).
+The architecture is depicted in the following figure:
+
+<p align="center">
+  <img src="./imgs/fabric_decoupled.png">
```

Comment:
Could you point me to the image, I can upload it and reference via https link here :)

Reply:
There you go  

![fabric_coupled](https://user-images.githubusercontent.com/18405289/225887472-6b64582f-1b2b-44a6-a707-bce80861bfa1.png)
![fabric_decoupled](https://user-images.githubusercontent.com/18405289/225887446-4d619cdc-8988-4b1e-89ab-7d1f447cd892.png)